mountpoint-s3-client/tests/endpoint.rs

```diff
@@ -88,3 +88,88 @@ async fn test_addressing_style_uri_fips_dualstack(addressing_style: AddressingSt
     })
     .await;
 }
+
+// Transfer acceleration can only be supported with virtual-hosted-style addressing
+#[test_case(AddressingStyle::Virtual)]
+#[tokio::test]
+async fn test_addressing_style_uri_transfer_acceleration(addressing_style: AddressingStyle) {
+    run_test(|_region| {
+        let uri = "https://s3-accelerate.amazonaws.com".to_string();
+        Endpoint::from_uri(&uri, addressing_style).unwrap()
+    })
+    .await;
+}
+
+async fn run_access_points<F: FnOnce(&str) -> Endpoint>(f: F) {
+    let sdk_client = get_test_sdk_client().await;
+    let (access_point, prefix) =
+        get_test_access_point_alias_and_prefix("test_access_point", AccessPointType::SingleRegion);
```

Comment:
Is this the only difference with `run_test`?  If so, it may be worth to refactor to merge them.